### ISSUE TITLE
allow for a vertical offset

### ISSUE THREAD

**Issue by dlockhart** [Brightspace/d2l-dropdown-ui#20]
See attached. In this particular example, the opener's height is required to be larger than it appears to be -- in this case so that the hover/focus effects can appear at the very top. So the opener is actually ~40px higher than the visual "drop" arrow.

To fix this, it would be awesome to be able to provide a vertical offset -- in this case negative -- that could be used to bump the dropdown up/down.

![screen shot 2016-05-25 at 12 12 25 pm](https://cloud.githubusercontent.com/assets/5491151/15550527/c974f498-227f-11e6-8aae-c32dae406b4d.png)

**Comment by capajon** [comment]
Created PR: https://github.com/Brightspace/d2l-dropdown-ui/pull/23

**Comment by dbatiste** [comment]
@capajon merged his stuff, so we should be able to use the offset attributes (current default is vertical: 20, horizontal: 0).
```
<d2l-dropdown ... horizontal-offset="50" vertical-offset="25">
```

**Comment by dlockhart** [comment]
Sounds good! @capajon: if you want to auto-close issues with a commit, you can include "fixes #20" in your commit comment, it's pretty neat!

**Comment by capajon** [comment]
Good to know, thanks!
